### DISCUSSION TITLE
Dell Optiplex/Precision Tower support?

### DISCUSSION THREAD

**Discussion by nfnhayden** [Rem0o/FanControl.Releases#586] [Ideas]
would it be possible to add in support for these towers? alot of my projects are older prebuilts and as far as i can tell i had no control of the cpu, powersupply, or case fan but did the GPU....

**Comment by Rem0o** [comment]
Lookup https://github.com/Rem0o/FanControl.DellPlugin.

Had a dell tower at work, don't remember the exact type, but it did work.

**Reply by nfnhayden** [reply]
I will try that.... I didn't see any plugins earlier... I'll report back and let you guys now how well it works...
